packages/generator/templates/app/src/app/auth/mutations/signup.ts

```diff
@@ -0,0 +1,18 @@
+import db from "db"
+import {SecurePassword} from "@blitzjs/auth/secure-password"
+
+export default async function signup(input: {password: string; email: string}, ctx: any) {
+  const blitzContext = ctx
+  const hashedPassword = await SecurePassword.hash((input.password as string) || "test-password")
+  const email = (input.email as string) || "test" + Math.random() + "@test.com"
```

Comment:
I assume this is WIP for testing?

Also, we use `input.email.toLowerCase().trim()` for some basic cleanup.

Reply:
oops, yeah it was for testing, will remove it, weird how it was missed

Reply:
done